packages/gluestick/src/cli/logger.js

```diff
@@ -13,28 +15,78 @@ const levels = {
   error: 40,
 };
 
-const loggerFactory = (type: string, level: string) => (...args) => {
+// If you need to find what module prints directly to console.log, uncomment this code
+// and replace `Put some text here` with some word from log you're interested in
+// console._log = console.log.bind(console);
+// console.log = (...args) => {
+//   console._log(...args);
+//   if (args[0].includes('Put some text here')) {
+//     console.trace('here');
+//   }
+// };
```

Comment:
This is a bit ugly. Probably some flag when you run `--dev` would be nicer? Although we can do this in the future, not necessary for this change.